### PR TITLE
Random mappings

### PR DESCRIPTION
**Pull request by BoogieMonster1O1**
stole some from legacy fabric :yeefrickinhaw:

### CODE REVIEW

File: mappings/net/minecraft/block/MetalBlock.mapping [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_55 net/minecraft/block/MetalBlock
```
Comment (sschr15) [inline mappings/net/minecraft/block/MetalBlock.mapping:1]: so we're calling diamonds a metal?

File: mappings/net/minecraft/client/entity/particle/ParticleEntity.mapping [hunk 2]
```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_262 net/minecraft/client/entity/particle/ParticleEntity
```
Comment (sschr15) [inline mappings/net/minecraft/client/entity/particle/ParticleEntity.mapping]: `Particle` or `ParticleEntity`? Other entities (with the exception of the player instance class) don't have the `Entity` suffix.

File: mappings/net/minecraft/item/Item.mapping [hunk 3]
```diff
@@ -77,7 +77,10 @@ CLASS net/minecraft/class_123 net/minecraft/item/Item
 	FIELD field_415 DIAMOND_PICKAXE Lnet/minecraft/class_123;
 	FIELD field_416 DIAMOND_AXE Lnet/minecraft/class_123;
 	METHOD method_310 onUse (Lnet/minecraft/class_127;Lnet/minecraft/class_101;Lnet/minecraft/class_152;)Lnet/minecraft/class_127;
-	METHOD method_311 onUseAtLocation (Lnet/minecraft/class_127;Lnet/minecraft/class_152;Lnet/minecraft/class_101;IIII)Z
+	METHOD method_311 useOnBlock (Lnet/minecraft/class_127;Lnet/minecraft/class_152;Lnet/minecraft/class_101;IIII)Z
```
Comment (sschr15) [inline mappings/net/minecraft/item/Item.mapping]: Should `onUse` be changed to `use` with this naming?

File: mappings/paulscode/sound/SoundSystem.mapping [hunk 4]
```diff
@@ -0,0 +1 @@
+CLASS paulscode/sound/SoundSystem paulscode/sound/SoundSystem
```
Comment (sschr15) [inline mappings/paulscode/sound/SoundSystem.mapping:1]: Why does this have a mapping? It's not obfuscated.

File: mappings/net/minecraft/world/BlockBox.mapping [hunk 5]
```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/class_156 net/minecraft/world/BlockBox
```
Comment (sschr15) [inline mappings/net/minecraft/world/BlockBox.mapping]: `BlockBox` or `Box`? Given that the positions are doubles, I believe that they can take up a different size than a block.

**Comment by PhoenixVX** [comment]
E

**Comment by IMS212** [comment]
thank you very insightful

File: mappings/net/minecraft/item/Item.mapping (hunk 3 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/item/Item.mapping]: sure will do

File: mappings/paulscode/sound/SoundSystem.mapping (hunk 4 above)
Comment (BoogieMonster1O1) [reply mappings/paulscode/sound/SoundSystem.mapping:1]: thing I ctrl oed it

File: mappings/net/minecraft/block/MetalBlock.mapping (hunk 1 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/block/MetalBlock.mapping:1]: yes

Comment (BoogieMonster1O1) [reply mappings/net/minecraft/block/MetalBlock.mapping:1]: it makes sense
MaterialBlock got me confused when I was mapping Material

File: mappings/net/minecraft/class_248.mapping [hunk 6]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_248
```
Comment (sschr15) [inline mappings/net/minecraft/class_248.mapping:1]: Empty class mapping?

File: mappings/net/minecraft/block/FallingBlock.mapping [hunk 7]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_78 net/minecraft/block/FallingBlock
```
Comment (sschr15) [inline mappings/net/minecraft/block/FallingBlock.mapping:1]: Due to both this and `FallingBlockEntity`, could a comment be added for what makes this different?

**Review by sschr15 — COMMENTED** [review]
(composed on mobile)

Overall, pretty good. Falling blocks seem to be a bit confusing. Player entity names are great.

File: mappings/net/minecraft/class_248.mapping (hunk 6 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/class_248.mapping:1]: yes

File: mappings/net/minecraft/block/FallingBlock.mapping (hunk 7 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/block/FallingBlock.mapping:1]: please note that for every change you request i add a bunch of more changes :tiny_potato:

**Review by sschr15 — APPROVED** [review]

**Pull request merged**